Services/AzineMailgunMailerService.php

```diff
@@ -151,4 +152,33 @@ public function  sendSpamComplaintNotification($eventId)
         }
         return $messagesSent;
     }
+
+    /**
+     * @param HetrixtoolsServiceResponse $response
+     * @throws \Exception
+     * @return int $messagesSent
+     */
+    public function sendBlacklistNotification(HetrixtoolsServiceResponse $response)
+    {
+        $messagesSent = 0;
+        $failedRecipients = [];
+
+        /** @var \Swift_Message $message */
+        $message = $this->mailer->createMessage();
+        $message->setTo($this->spamAlertsRecipientEmail)
+            ->setFrom($this->fromEmail)
+            ->setSubject($this->translator->trans('notification.blacklist_received'))
+            ->setBody(
```

Comment:
the txt/plain part should be added too `$message->addPart($textBody, 'text/plain');`